### DISCUSSION TITLE
secure the sensor and the webinterface

### DISCUSSION THREAD

**Discussion by ritchie-w** [jomjol/AI-on-the-edge-device#1253] [Q&A]
Hi,
What about to secure the Web interface.
Best regards
Ritchie

**Comment by jomjol** [comment]
Hi Ritchie,
this is a several time discussed topic. We have decided not to implement this in this public version. Either you would do it right and this would mean, not only to implement something really well designed, but to also maintain it and update it with respect to new security issues etc.
Or: we don't implement anything and therefore make sure, that it is in the hand of the user to limit the access to the tool via VPN, dedicated network, physical safety measure ...

Therefore there is no plan to implement it currently. Sorry for not having a better forecast.
